lib/service/connect.go

```diff
@@ -117,6 +117,9 @@ func (process *TeleportProcess) reconnectToAuthService(role types.SystemRole) (*
 			process.log.Error("Can not join the cluster as node, the token expired or not found. Regenerate the token and try again.")
 		case connectErr != nil:
 			process.log.Errorf("%v failed to establish connection to cluster: %v.", role, connectErr)
+			if process.Config.Version == defaults.TeleportConfigVersionV3 && process.Config.ProxyServer.IsEmpty() {
+				process.log.Errorf("Check to see if the config has auth_server pointing to a Teleport Proxy. If it does, use proxy_server instead of auth_server.")
```

Comment:
Is it possible to inspect the error to see if it's likely we're trying to connect to a proxy? As it stands we have no idea here, would be nice to be reasonably sure that's the problem before recommending `proxy_server`

Reply:
I spoke with @strideynet about this, and we came to the conclusion that it's probably more hassle to check the error than not. TLS libraries on macOS emit different errors than on Linux, which would add complexity to the error check.

Reply:
that's fine, I wasn't sure how hard/easy it would be. I will just recommend a bit softer wording

Reply:
wait, I misread it the first time, this looks fine